framework/util/file_path.cpp

```diff
@@ -435,6 +435,30 @@ void GetApplicationInfo(FileInfo& file_info)
     {
         GFXRECON_LOG_WARNING("Failed to retrieve the application executable name");
     }
+#elif defined(__ANDROID__) || defined(__linux__)
+    std::ifstream cmdlineFile("/proc/self/cmdline");
+
+    if (cmdlineFile.is_open())
+    {
+        std::string line;
+        std::getline(cmdlineFile, line);
+
+        size_t pos = std::string::npos;
+        // Command line arguments separated by null bytes
+        if ((pos = line.find('\0')) != std::string::npos)
```

Comment:
A string is null-terminated by definition so this will always find the end of the string; can you tell me more about what you intended here?

Reply:
Oh! I got confused with `/proc/self/cmdline`. While reading it, arguments are separated with null characters. So I kinda tried to get the first argument by reading before the null character but probably when reading with getline it automatically copies the first argument only.

Let me fix it. This will also remove fix the other issue.

Reply:
Ptal